packages/client/src/clients/in-app-messages/message-client.ts

```diff
@@ -14,10 +15,14 @@ import {
 
 /**
  * Manages realtime connection to in app messages service.
+ *
+ * TODO: Rename to InAppMessagesClient to singular?
```

Comment:
This was feedback from Chris to match the other clients like `UsersClient` and `Preferences` although there are several that are singular. No strong opinion either way